tests.yaml

```diff
@@ -38,15 +38,15 @@ tests:
     values: '32768+0x61'
   alert_rule_test:
   - eval_time: 1h
-    alertname: KubePersistentVolumeFullInFourDays
+    alertname: KubePersistentVolumeFillingUp
```

Comment:
Since the alertname changed, this test now covers two alerts instead of one. This means test case needs to be adjusted with different values to prevent `critical` alert from firing.

Test output:
```
  FAILED:
    alertname:KubePersistentVolumeFillingUp, time:1h0m0s, 
        exp:"[]", 
        got:"[Labels:{alertname=\"KubePersistentVolumeFillingUp\", job=\"kubelet\", namespace=\"monitoring\", persistentvolumeclaim=\"somepvc\", severity=\"critical\"} Annotations:{message=\"The PersistentVolume claimed by somepvc in Namespace monitoring is only 1.294% free.\", runbook_url=\"https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-kubepersistentvolumefillingup\"}]"
```

Reply:
I did it a bit differently to the same effect hopefully, please take a look again, thanks!

Reply:
We should also add a negative test to check if alert is not firing.